### DISCUSSION TITLE
No documentation for git email and username causes actions to use 41898282+github-actions[bot]@users.noreply.github.com

### DISCUSSION THREAD

**Discussion by alex-page** [community/community#119597] [Actions]
### Select Topic Area

Product Feedback

### Body

There is this weird thing when you need to run `git` commands in a GitHub action and you need to provide an email and username. Searching the internet, community discussions and code across GitHub the following email and username are recommend:
- email: `41898282+github-actions[bot]@users.noreply.github.com`
- username: `github-actions[bot]`

There is no official documentation on this account or email. It shows up over [15k times when doing a code search](https://github.com/search?q=41898282%2Bgithub-actions%5Bbot%5D%40users.noreply.github.com&type=code). Trying to explain to other developers why this email and username are used is not clear. Often developers want to run GitHub actions and git without the commits being tied back to their account.

This has [been brought up as a solution in multiple discussions](https://github.com/orgs/community/discussions?discussions_q=41898282%2Bgithub-actions%5Bbot%5D%40users.noreply.github.com) but never documented. If possible it would be great to have a recommended `username` and `email` in the documentation.

**Comment by XVII** [comment]
It's a de facto standard, but it should be documented.

The [**github-actions[bot]** user](https://api.github.com/users/github-actions%5Bbot%5D) has been around since 2018 with the user `type` of `bot`. The user's URL is https://github.com/apps/github-actions, which redirects to the [official GitHub Actions page](https://github.com/features/actions). That's the most "official" link I could find.

**Comment by codytodonnell** [comment]
I am encountering this same issue in September 2026. I would appreciate some official documentation around this specific convention.

**Comment by apekshhaa** [comment]
Same issue here in September 2026. Would be great if GitHub officially documented this convention. I kept finding 41898282+github-actions[bot]@users.noreply.github.com everywhere, but couldn’t find an official explanation for why this is the correct identity to use.
